lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -154,6 +154,12 @@ def publishing_api_has_item(item)
         stub_request(:get, url).to_return(status: 200, body: item.to_json, headers: {})
       end
 
+      def publishing_api_has_links(links)
+        links = links.with_indifferent_access
+        url = PUBLISHING_API_V2_ENDPOINT + "/links/" + links[:content_id]
```

Comment:
Would `content_id` be in the links payload? Perhaps this should be passed in as an arg to the function.